kuma/javascript/src/header/header.jsx

```diff
@@ -213,6 +213,47 @@ export default function Header(props: Props): React.Node {
             <div css={styles.login}>
                 <Login />
             </div>
+
+            {/* Remove this div when we're done asking for feedback */}
```

Comment:
Thinking ahead, a fear is that in a couple of months, when we need to remove this little flap, we start here where the React element code is and then have to work backwards to figure out what else to change. 
Can you please mention this URL `https://github.com/mozilla/kuma/pull/5511/files` in that comment since the instruction should effectively be to **revert this PR**.